q2_viromics/genomad_analysis.py

```diff
@@ -104,24 +98,23 @@ def genomad_analysis(
                 min_number_genes,
                 conservative_taxonomy,
             )
-
             # Define the filenames and destination paths in a list of tuples
             files_and_destinations = [
                 (
                     f"{id}_contigs_summary/{id}_contigs_virus.fna",
-                    viral_path,
+                    os.path.join(str(viruses), f"{id}_contigs.fa"),
                 ),
                 (
                     f"{id}_contigs_find_proviruses/{id}_contigs_provirus.fna",
-                    proviral_path,
+                    os.path.join(str(proviruses), f"{id}_contigs.fa"),
                 ),
                 (
                     f"{id}_contigs_summary/{id}_contigs_plasmid.fna",
-                    plasmids_path,
+                    os.path.join(str(plasmids), f"{id}_contigs.fa"),
                 ),
                 (
                     f"{id}_contigs_summary/{id}_contigs_virus_summary.tsv",
-                    virus_summary_path,
+                    os.path.join(str(virus_summary), f"{id}_virus_summary.tsv"),
```

Comment:
But now you have a mix of manual with string "/" and os.path.join. 
Do either the one or the other in all paths for consistency.